Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:18-alpine AS builder
+
+WORKDIR /app
+
+COPY package*.json ./
+COPY tsconfig.json ./
+COPY public ./public
+COPY src ./src
+
+RUN npm install
+
+RUN npm run build
+
+FROM nginx:alpine
+
+COPY --from=builder /app/build /usr/share/nginx/html
+
+RUN rm /etc/nginx/conf.d/default.conf
+COPY nginx/nginx.conf /etc/nginx/conf.d
+
+EXPOSE 80
```

Comment:
Skąd taka wartość portu? Na takim się stawia aplikacje frontendowe?

Reply:
W goole `nginx port` : 
By default, NGINX Ingress Controller listens on ports 80 and 443. These ports can be changed easily, but modifying the listen ports for your NGINX Ingress resources will require the editing of . tmpl files.

Mogę zmienić na 3000.

Reply:
Jeśli tak jest domyślnie to niech zostanie